### DISCUSSION TITLE
Updating

### DISCUSSION THREAD

**Discussion by Brantlab** [HiEventsDev/Hi.Events#53] [General]
What is the best way to update? Repull the git down then move my docker compose back into it or ?

**Comment by daveearley** [comment]
Yes, assuming you're using the development Docker setup, you can pull the latest changes then run the [`./start-dev.sh`](https://github.com/HiEventsDev/hi.events/blob/42d80a629002e3a19dbc85f5ab66ee86dc8dda32/docker/development/start-dev.sh) again. That should run any new migrations and restart the containers.

**Reply by Brantlab** [reply]
I ended up 
```
cp docker-compose.yml docker-compose.yml.back
rm docker-compose.yml
git pull 
cp docker-compose.yml.back docker-compose.yml
docker compose up -d --build
```
This ended up working well. I am working from the all-in-one folder.
